Added initial data model

mujahidkhan17 opened the pull request:
Created DataModel and
Job Controller with Get Method for Jobs and skeleton for Post, Put and Delete, 
Users Controller with Get Method for Recruiters

mujahidkhan17 committed:
Added initial data model

ubaid1900 reviewed (commented):
Looks good. Add the bin and obj folder to git ignore. We don't need them in source control.

mujahidkhan17 committed:
Recruites solution in backend folder,
Add project for Cosmos db creation for app
Db created, Insert not working

mujahidkhan17 committed:
Add solution files in BackEnd Folder,
Created Solution for Cosmos Db creation for application,
DB Created - Insert is not working.

ubaid1900 committed:
Working console app.

mujahidkhan17 committed:
Inserting Jobs into DB using Jobs List

mujahidkhan17 committed:
Azure cosmos db context  created --> connect to container Jobs and Get data

mujahidkhan17 committed:
Addeded AddScoped to activate controllers --> DB Connectivity, API  with Get all Jobs method working

mujahidkhan17 committed:
Add Scoped to activate Controllers and API to get all jobs method working

mujahidkhan17 committed:
Job Controller skeleton with CRUD operations completed

mujahidkhan17 committed:
Job Controller skeleton with CRUD operations completed
